apps/website/src/utils/image-converter.ts

```diff
@@ -0,0 +1,91 @@
+import type { Dispatch, SetStateAction } from "react";
+
+async function convertHeicToJpeg(file: File) {
+  const heic2any = (await import("heic2any")).default;
+  const blob = await heic2any({
+    blob: file,
+    toType: "image/jpeg",
+    quality: 0.9,
+  });
+  return new File([blob as Blob], file.name.replace(/\.[^.]+$/, "jpeg"), {
```

Comment:
```suggestion
  return new File([blob as Blob], file.name.replace(/\.[^.]+$/, ".jpeg"), {
```
Your regex pattern includes the `.` so I imagine you want it in the replacement string too?